sdk/inc/azure/core/az_mqtt5_rpc.h

```diff
@@ -110,6 +110,60 @@ typedef enum
   AZ_MQTT5_RPC_STATUS_TIMEOUT = 504,
 } az_mqtt5_rpc_status;
 
+/**
```

Comment:
Please keep internal functions in internal headers. (e.g. follow the pattern from `az_config_internal.h`).